addons/mail/static/src/core/common/thread_model.js

```diff
@@ -325,6 +325,10 @@ export class Thread extends Record {
         return ["chat", "group"].includes(this.type);
     }
 
+    get isSelfCall() {
+        return this.channelMembers.length === 1 && this.channel_type === "chat";
+    }
```

Comment:
Move this part to `Discuss` folder as it is only used for discuss app and maybe rename the getter to something more relatable.